### ISSUE TITLE
Como mantener código legacy

### ISSUE THREAD

**Issue by maxcodes** [MexicoOnRails/platicas#12]
Ya hemos tenido algunas charlas de esto, pero nunca son suficientes 😅  Se buscan ejemplos de cómo hacer refactoring, y aún mejor, cómo escribir el código de forma que sea fácil de mantener en el futuro 👼

**Comment by alosada** [comment]
o/
Les puedo mostrar un caso de refactoring de donadora :)

**Comment by maxcodes** [comment]
Sí!! Estaría genial. Este sería un gran tema para el siguiente meetup (14 mayo). Te anotas?

**Comment by alosada** [comment]
@maxcodes el 14 de Mayo no se si pueda ir, me voy de viaje al dia siguiente, pero en Junio si me anoto.
